app/controllers/media_objects_controller.rb

```diff
@@ -410,31 +410,25 @@ def destroy
   # it will just toggle the state.
   def update_status
     status = params[:status]
-    errors = []
-    success_count = 0
+    errors, update_errors = [], [], []
```

Comment:
Also one too many `[]`

Reply:
Same here codeclimate complained when there are equal number of `[]`s